README.md

```diff
@@ -11,15 +11,24 @@ A guide for front-end development at the [CFPB](http://cfpb.github.io/).
 - [CSS/Less standards](css.md)
 - [JavaScript standards](javascript.md)
 
-## Guidelines
+## Capital Framework
+
+Capital Framework is our modular front end framework.
+
+- [Capital Framework docs](http://cfpb.github.io/capital-framework/)
+- [cf-generator](https://github.com/cfpb/generator-cf)
+
+## Guides
 
 - [Publishing node modules](npm.md)
```

Comment:
Should this be ordered in how you might need them? IE build your project, review your code, publish your code?